index.bs

```diff
@@ -910,7 +910,7 @@ dictionary MLOperandDescriptor {
     1. Let |elementLength| be 1.
     1. [=map/For each=] |dimension| of |desc|.{{MLOperandDescriptor/dimensions}}:
         1. Set |elementLength| to |elementLength| × |dimension|.
-    1. Let |elementSize| be the [=element size=] of one of the {{ArrayBufferView}} types that matches |desc|.{{MLOperandDescriptor/type}} according to [this table](#appendices-mloperandtype-arraybufferview-compatibility).
+    1. Let |elementSize| be the [=element size=] of one of the {{ArrayBufferView}} types that matches |desc|.{{MLOperandDescriptor/type}} according to [this table](#appendices-mloperanddatatype-arraybufferview-compatibility).
```

Comment:
@fdwr Here `MLOperandDescriptor/type` should be `MLOperandDescriptor/dataType`, and please update the others below, thanks.

Reply:
Good catch!

Reply:
Ooh sneaky, using `/` instead of .type, :type, type:: datatype- ...